src/probnum/filtsmooth/gaussfiltsmooth/extendedkalman.py

```diff
@@ -32,8 +37,11 @@ def transition_realization(self, real, start, stop, **kwargs):
             dispmatfun=self.non_linear_sde.dispersionmatrix,
         )
 
-    def transition_rv(self, rv, start, stop, **kwargs):
-        jacobfun = functools.partial(self.non_linear_sde.jacobian, state=rv.mean)
+    def transition_rv(self, rv, start, stop, linearise_at=None, **kwargs):
+        if linearise_at is None:
+            jacobfun = functools.partial(self.non_linear_sde.jacobian, state=rv.mean)
+        else:
+            jacobfun = functools.partial(self.non_linear_sde.jacobian, state=rv.mean)
```

Comment:
You didn't actually linearise at the `linearise_at` spot ;)

Reply:
oooops^^